src/simulation.py

```
@@ -0,0 +1,241 @@
+        customer_id = 1
+        
+        while True:
+            # Wait for next customer arrival
+            interarrival_time = random.expovariate(1.0 / self.config.interarrival_time)
+            yield env.timeout(interarrival_time)
+            
+            # Record arrival time
+            arrival_time = env.now
+            
+            # Randomly choose customer type (70% in-house, 30% food app)
+            if random.random() < 0.7:
```

Ruff (0.13.1)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)